blocks/bluesection/bluesection.css

```diff
@@ -0,0 +1,40 @@
+
+
+.bluesection-container .bluesection-wrapper {
+    max-width: unset;
+    padding: 0;
+  }
+
+.blue-section {
+    padding: 20px;
+    background-color: #388ce1;
+    color: white;
+    text-align: center;
+}
+
+.blue-section h4 {
+    font-family: franklin-heavy-italic, sans-serif;
+    font-size: 1.625rem;
+    margin: 0;
+}
+
+.blue-section p {
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 300;
+    font-size: 1rem;
+    margin: 0 0 20px;
+}
+
+.blue-section p a {
+    color: white;
+    text-decoration: underline;
+    font-style: italic;
+    font-family: 'Open Sans', sans-serif !important;
```

Comment:
we need to create a font-family for Open Sans, and that also includes a Fallback.

Additionally, `!important!` should not be needed in any context